src/entities/trades/swapr-v3/constants.ts

```diff
@@ -0,0 +1,56 @@
+export const POOL_DEPLOYER_ADDRESS = '0xC1b576AC6Ec749d5Ace1787bF9Ec6340908ddB47'
+export const POOL_INIT_CODE_HASH = '0xbce37a54eab2fcd71913a0d40723e04238970e7fc1159bfd58ad5b79531697e7'
+
+export const SWAPR_ALGEBRA_CONTRACTS = {
+  quoter: '0xcBaD9FDf0D2814659Eb26f600EFDeAF005Eda0F7',
+  router: '0xfFB643E73f280B97809A8b41f7232AB401a04ee1',
+}
+
+export type BaseToken = {
+  chainId: number
+  decimals: number
+  symbol: string
+  name: string
+  isNative: boolean
+  isToken: boolean
+  address: string
+}
+
+export const baseTokens: BaseToken[] = [
+  {
+    chainId: 100,
```

Comment:
We can use `ChainId.GNOSIS` here. The same for other values. But you need to update the type so this can work.

Reply:
updated

Reply:
But I think here is not that important to use ChainId.Gnosis constant, but I'll take it.